int32 iamready
---
int32 received
int32 numFound
apriltags_ros/AprilTagDetectionArray detections_all
int32 numRequested
apriltags_ros/AprilTagDetectionArray detections_tograb
geometry_msgs/PoseStamped robotPose
